runner/execer/fake/intercept_test.go

```diff
@@ -0,0 +1,48 @@
+package fake
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/scootdev/scoot/runner/execer"
+)
+
+func TestInterceptor(t *testing.T) {
+	var wg1, wg2 sync.WaitGroup
+	ex1, ex2 := NewSimExecer(&wg1), NewSimExecer(&wg2)
+	ex := &InterceptExecer{
+		Condition:   StartsWithSimExecer,
+		Interceptor: ex1,
+		Default:     ex2,
+	}
+
+	// block1 will block on wg1; block2 on wg2
+	block1 := execer.Command{
+		Argv: []string{"#! sim execer", "pause", "complete 0"},
+	}
+	block2 := execer.Command{
+		Argv: []string{"pause", "complete 0"},
+	}
+
+	wg2.Add(1)
```

Comment:
why are you using sync wait groups?  There aren't any go routines being executed right?  Everything is running synchronously in this test, or am I missing something?